docs/installation.rst

```diff
@@ -4,10 +4,41 @@
 Installation
 ============
 
+Getting the dependencies
+------------------------
+
+We suggest you use `anaconda`_ (or `miniconda`_) with Python 3.7 for the
+installation. However, you can also get the dependencies through your prefered
+package manager.
+
+Using `conda` you can create an isolated environment for KaSKA.
+
+.. code-block:: console
+
+   $ conda config --add channels conda-forge
+   $ conda create -n kaska python=3.7 tensorflow numba gdal
```

Comment:
version of gdal

Reply:
Tensorflow maybe smaller than 2.

Reply:
Presumably, as per issue #87, we need to set maximum version of `goal`?
i.e. `gdal"<=2.5.2"`

Reply:
See PR #91